jcl/src/java.base/share/classes/java/lang/invoke/VarHandle.java

```diff
@@ -1692,5 +1733,32 @@ MethodHandle getMethodHandle(int i) {
 	}
 /*[ENDIF] Java15 | OPENJDK_METHODHANDLES */
 
-	abstract MethodType accessModeTypeUncached(AccessMode accessMode);
+	MethodType accessModeTypeUncached(
+		/*[IF Java16]*/
+		AccessType type
+		/*[ELSE]*/
+		AccessMode accessMode
+		/*[ENDIF] Java16 */
+	) {
+		throw OpenJDKCompileStub.OpenJDKCompileStubThrowError();
+	}
+
+/*[IF Java16]*/
+	final MethodType accessModeTypeUncached(int index) {
+		return accessModeTypeUncached(AccessType.values()[index]);
+	}
+
+	public VarHandle withInvokeExactBehavior() {
+		throw OpenJDKCompileStub.OpenJDKCompileStubThrowError();
+	}
+
+	public VarHandle withInvokeBehavior() {
```

Comment:
`withInvokeExactBehavior` & `withInvokeBehavior` are public API, they are `abstract` as per https://download.java.net/java/early_access/jdk16/docs/api/java.base/java/lang/invoke/VarHandle.html#withInvokeBehavior()

Reply:
No impact seen on the JCK `api/signaturetest`. More details provided through DM.

Reply:
Java 16 builds are not fully supported yet. So, it is not possible to verify the impact of changing `abstract` methods to `concrete` methods in the `VarHandle` class. These changes should not break any OpenJ9 testing so they can be accepted in its current state. Any other issues can be addressed once Java 16 builds are fully functional.